kpack-image-builder/config/manager/kustomization.yaml

```diff
@@ -13,6 +13,6 @@ configMapGenerator:
   name: manager-config
 
 images:
-  - name: cloudfoundry/korifi-kpack-image-builder
-    newName: cloudfoundry/korifi-kpack-image-builder
-    newTag: latest
+- name: cloudfoundry/korifi-kpack-image-builder
+  newName: korifi-kpack-image-builder
+  newTag: 122DDD51-2036-4A7B-8234-CED0D5879E6D
```

Comment:
This is why e2es are failing 🙂

Reply:
@gcapizzi is this safe to commit to main? 
I see that your [PR](https://github.com/cloudfoundry/korifi/pull/1141) doesn't need it?

Reply:
Yeah my PR is unrelated. I thought this change didn't belong here as it removes the `cloudfoundry/` prefix from `newName` (which is the reason of the failure) + it sets `newTag` to some random ID instead of `latest`.